.github/workflows/release.yml

```diff
@@ -51,29 +47,28 @@ jobs:
         with:
           repository: 'neovim/neovim'
           fetch-depth: 0
-      - run: |
-         apt-get update
-         apt-get install -y build-essential curl gettext ninja-build unzip
+
+      # zig cc with -02 implicitly adds -DNDEBUG so remove this from the generator flags
+      - name: Patch Neovim
+        run: sed -i '/APPEND gen_cflags -O2/d' src/nvim/CMakeLists.txt
```

Comment:
You can remove this after merging the PR on neovim repo

Reply:
We'll need it to respin 0.10

Reply:
Huh, fair. Backportbthe neovim pr?